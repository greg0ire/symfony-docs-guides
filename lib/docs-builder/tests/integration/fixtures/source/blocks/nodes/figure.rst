.. figure:: images/logo.png
   :alt: Symfony Logo
   :width: 200px

I am a paragraph AFTER the figure. I should not be included as the
caption for the above figure.

.. figure:: images/logo.png

    But I am a caption *for* the figure above.

Some images use a special CSS class to wrap a fake browser around them:

.. image:: images/exceptions-in-dev-environment.png
   :alt: A typical exception page in the development environment
   :align: center
   :class: some-class with-browser another-class

And RST figures use a different syntax to define their custom CSS classes:

.. figure:: images/logo.png
    :alt: /
    :align: center
    :figclass: with-browser foo
