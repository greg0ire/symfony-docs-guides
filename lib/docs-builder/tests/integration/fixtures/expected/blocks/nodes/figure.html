<figure>
    <!-- REMOVE: src is empty just because the test doesn't utilize the full stack, with CopyImagesListener -->
    <img src="" alt="Symfony Logo" width="200px" />
</figure>
<p>I am a paragraph AFTER the figure. I should not be included as the
caption for the above figure.</p>

<figure>
    <img src="" />

    <figcaption><p>But I am a caption <em>for</em> the figure above.</p></figcaption>
</figure>

<p>Some images use a special CSS class to wrap a fake browser around them:</p>
<div class="with-browser">
    <img src="" alt="A typical exception page in the development environment" align="center" class="some-class with-browser another-class">
</div>

<p>And RST figures use a different syntax to define their custom CSS classes:</p>

<figure class="with-browser foo">
    <img src="" alt="/" align="center">
</figure>
